en/identity-server/7.0.0/mkdocs.yml

```diff
@@ -636,12 +641,12 @@ nav:
           - Network-level: deploy/security/security-guidelines/network-level-security-guidelines.md
         - Encryption:
           - Asymmetric encryption:
-            - Use Asymmetric encryption: deploy/security/asymmetric-encryption/use-asymmetric-encryption.md
-            - Create new keystores: deploy/security/asymmetric-encryption/create-new-keystores.md
-            - Manage keystores: deploy/security/asymmetric-encryption/configure-keystores-in-wso2-products.md
-            - Renew a CA-Signed certificate in a keystore: deploy/security/asymmetric-encryption/renew-a-ca-signed-certificate-in-a-keystore.md
-            #- Manage Keystores via UI: deploy/security/asymmetric-encryption/manage-keystores-with-the-ui.md
-            - Add multiple keys to the primary keystore: deploy/security/asymmetric-encryption/add-multiple-keys-to-the-primary-keystore.md
+            - Asymmetric encryption: deploy/security/asymmetric-encryption/index.md
+          - Keystores:
```

Comment:
move the keystores section to the third position.

- symmetric
- asymmetric
- keystores